assets/appmanifest.json

```diff
@@ -0,0 +1,64 @@
+{
+    "$schema":"https://developer.microsoft.com/en-us/json-schemas/teams/v1.19/MicrosoftTeams.schema.json",
+    "version":"{{.AppVersion}}",
+    "manifestVersion":"1.19",
+    "id":"{{.AppID}}",
+    "name":{
+       "short":"{{.AppName}}",
+       "full":"{{.AppName}}"
+    },
+    "developer":{
+       "name":"Mattermost",
+       "mpnId":"",
+       "websiteUrl":"https://github.com/mattermost/mattermost-plugin-msteams-devsecops/",
+       "privacyUrl":"https://mattermost.com/privacy-policy/",
+       "termsOfUseUrl":"https://mattermost.com/software-services-license-agreement/"
+    },
+    "description":{
+       "short":"Connect with Mattermost for seamless team collaboration.",
+       "full":"The Mattermost app for Microsoft 365 provides seamless access to the Mattermost platform from inside the tools you use every day. Communicate and collaborate effectively with your team in a centralized and familiar interface.\n\nKey Features:\n- Direct Access: Access Mattermost directly from a tab without switching applications or opening a browser\n- Seamless Integration: Experience the full functionality of Mattermost within a familiar interface\n- Real-time Collaboration: Collaborate with your team on projects, workflows, and communications in real time\n\nThis app is designed to work with Microsoft 365, Outlook, and Microsoft Teams. An account with Mattermost is required to use this app.\n\nAbout Mattermost: \nMattermost is a purpose-built platform for technical and operational teams working in organizations vital to national security, public safety, and critical infrastructure. [https://mattermost.com/](https://mattermost.com/)"
+    },
+    "icons":{
+       "outline":"outline.png",
+       "color":"color.png"
+    },
+    "accentColor":"#1e325c",
+    "staticTabs":[
+       {
+          "entityId":"main",
+          "name":"Mattermost",
+          "contentUrl":"https://{{.SiteDomainPath}}/plugins/{{.PluginID}}/iframe/mattermostTab",
+          "scopes":[
+             "personal"
+          ],
+          "context":[
+             "personalTab"
+          ]
+       },
```

Comment:
We need to remember to add the notification preview tab here when the logic is migrated to this plugin.

Reply:
@fmartingr I added it.  PHAL.